platform/fio/transaction.go

```diff
@@ -2,8 +2,78 @@ package fio
 
 import (
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/errors"
+	"encoding/json"
+	"strconv"
+	"strings"
+	"time"
 )
 
 func (p *Platform) GetTxsByAddress(address string) (page blockatlas.TxPage, err error) {
-	return page, err
+	account := address // TODO this should be actor
+	actions, err := p.client.getTransactions(account)
+	if err != nil {
+		return nil, err
+	}
+	var txs []blockatlas.Tx = make([]blockatlas.Tx, 0)
+	for _, a := range actions {
+		tx, err := p.Normalize(&a, account)
+		if err != nil {
+			continue
+		}
+		txs = append(txs, tx)
+	}
+	txPage := blockatlas.TxPage(txs)
+	return txPage, nil
+}
+
+func (p *Platform) Normalize(action *Action, account string) (blockatlas.Tx, error) {
+	var from string
```

Comment:
var (
  from, to string
  amount blockatlas.Amount
)

Reply:
done